providers/vsphere/resources/vsphere.lr.manifest.yaml

```diff
@@ -105,6 +109,16 @@ resources:
       name:
       - vmware-esxi
       - vmware-vsphere
+  platform:
```

Comment:
I do not understand why these are showing up here. They aren't part of the `vsphere.lr` file.

Reply:
They need to be removed here